src/Components/Login.js

```diff
@@ -14,17 +14,22 @@ const Login = () => {
     e.preventDefault();
 
     try {
-      const response = await axios.post("http://localhost:8080/api/v1/auth/login", input, {
-        headers: {
-          "Content-Type": "application/json",
-        },
-      });
+      const response = await axios.post(
+        "http://localhost:8080/api/v1/auth/login",
```

Comment:
It would be better to add the baseUrl and the url in constants file

Reply:
added the baseURL and .env file for configuration